### ISSUE TITLE
Consumption_Standard : Save and Discard button is still enable after saving the workflow.

### ISSUE THREAD

**Issue by vgouth** [Azure/LogicAppsUX#1999]
### Describe the Bug with repro steps

1.Create a Logic apps in consumption / standard add stateful or stateless workflow.
2.Create a workflow into it.
3.Try to save the workflow.
4.After saving the workflow, Save and Discard button should be disabled.

**Expected :** Save and discard button should be disabled after saving the workflow.

**Actual :** Save and Discard button is still enabled after saving the workflow.

### Workflow JSON

```json
{
  "definition": {
    "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
    "actions": {
      "Response": {
        "type": "Response",
        "kind": "Http",
        "inputs": {
          "statusCode": 200,
          "body": "@{parameters('b')}@{parameters('a')}"
        },
        "runAfter": {},
        "trackedProperties": {}
      }
    },
    "contentVersion": "1.0.0.0",
    "outputs": {},
    "triggers": {
      "When_a_HTTP_request_is_received": {
        "type": "Request",
        "kind": "Http"
      }
    }
  },
  "connectionReferences": {},
  "parameters": {
    "b": {
      "type": "Bool",
      "value": true
    },
    "a": {
      "type": "Int",
      "value": 1
    }
  }
}
```

### Screenshots or Videos

![image](https://user-images.githubusercontent.com/113517384/231772933-aa53bbef-f728-4c57-9ac5-0675d3a01288.png)
![image](https://user-images.githubusercontent.com/113517384/231773255-22bd80e3-d18e-4b57-a687-6008d200eaa8.png)

### Browser

Edge

### Additional context

**Environmental Details:**

**URL:** [https://aka.ms/emastaging-flowstip1](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faka.ms%2Femastaging-flowstip1&data=05%7C01%7Cv-govubilish%40microsoft.com%7C5dc1b66779e54ebfb20b08db3bf92e3f%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638169713681866036%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Ox8FGZYcViENAadUWbkyOt05VGv%2FAvR7SYpEG%2BGHVDI%3D&reserved=0) 
**Version :** 2.30410.1.3

**Comment by hartra344** [comment]
@vgouth this is by design in the new designer

**Comment by Eric-B-Wu** [comment]
yep we have dirty state tracking now, so it should not be able to handle the state of the workflow
